SPECS/kernel/config_aarch64

```diff
@@ -2787,14 +3902,25 @@ CONFIG_VT_CONSOLE_SLEEP=y
 CONFIG_HW_CONSOLE=y
 CONFIG_VT_HW_CONSOLE_BINDING=y
 CONFIG_UNIX98_PTYS=y
-# CONFIG_LEGACY_PTYS is not set
-# CONFIG_SERIAL_NONSTANDARD is not set
-# CONFIG_NOZOMI is not set
-# CONFIG_N_GSM is not set
-# CONFIG_TRACE_SINK is not set
-# CONFIG_NULL_TTY is not set
-# CONFIG_LDISC_AUTOLOAD is not set
-# CONFIG_DEVMEM is not set
+CONFIG_LEGACY_PTYS=y
```

Comment:
This config has been identified by the kernel config hardening checker and should be disabled

Reply:
updated.